claat/render/md.go

```diff
@@ -232,3 +233,37 @@ func (mw *mdWriter) header(n *types.HeaderNode) {
 		mw.writeBytes(newLine)
 	}
 }
+
+func (mw *mdWriter) grid(n *types.GridNode) {
+	mw.newBlock()
+	mw.writeString(`<table markdown="1">`)
+	for _, r := range n.Rows {
+		mw.writeString("\n<tr>")
+		for _, c := range r {
+			mw.writeFmt(`<td colspan="%d" rowspan="%d">`, c.Colspan, c.Rowspan)
+			mw.write(c.Content.Nodes...)
+			mw.writeString("</td>")
+		}
+		mw.writeString("\n</tr>")
+	}
+	mw.writeString("</table>")
+}
+
+func (mw *mdWriter) infobox(n *types.InfoboxNode) {
+	mw.newBlock()
+	mw.writeString(`<aside markdown="1" class="`)
+	mw.writeString(string(n.Kind))
+	mw.writeString(`">`)
+	mw.write(n.Content.Nodes...)
+	mw.writeString("</aside>")
+}
+
+func (mw *mdWriter) youtube(n *types.YouTubeNode) {
+	mw.newBlock()
+	mw.writeString(`<div class="video-wrapper">\n`)
+	mw.writeString(`  <iframe class="devsite-embedded-youtube-video" `)
+	mw.writeString(`data-autohide="1" data-showinfo="0" frameborder="0" `)
+	mw.writeString(`allowfullscreen data-video-id="`)
+	mw.writeString(string(n.VideoID))
```

Comment:
better yet, merged with the line above:

```go
mw.writeFmt("data-video-id=%q", n.VideoID)
```

in which case I would also move "allowfullscreen" part to the line above it.